sharding/sharding.go

```diff
@@ -172,6 +172,8 @@ func (r *ShardingFerry) Run() {
 
 	r.Ferry.Throttler.SetDisabled(false)
 
+	r.Ferry.StopTargetVerifier()
```

Comment:
This new call is essential for all Ghostferry based application now? It's not super intuitive imo. We should update or docs then.

Reply:
Planning to update the docs, yes. Unfortunately, there isn't really a better option as we need this to continue running even after `Ferry.Run()` completes. We need to extend the API of the Ferry somehow to stop the TargetVerification.